e2e/protractor.excludes.json

```diff
@@ -7,5 +7,7 @@
   "C260377": "https://alfresco.atlassian.net/browse/ACS-4467",
   "C260375": "https://alfresco.atlassian.net/browse/ACS-4467",
   "C286290": "https://alfresco.atlassian.net/browse/ACS-4467",
-  "C286472": "https://alfresco.atlassian.net/browse/ACS-4467"
+  "C286472": "https://alfresco.atlassian.net/browse/ACS-4467",
+  "C260387": "https://alfresco.atlassian.net/browse/ACS-4595",
```

Comment:
@alep85 you probably don't need to exluce these e2e anymore. From today they are success, see my PR: 
https://github.com/Alfresco/alfresco-ng2-components/pull/8240

@AnkurChauhan-131 noticed that they work from today. Please remove them from exclude.